keyboards/massdrop/ctrl/matrix.c

```diff
@@ -80,27 +83,29 @@ void matrix_init(void)
 }
 
 uint64_t mdebouncing = 0;
+bool debouncing = false;
+
 uint8_t matrix_scan(void)
```

Comment:
This doesn't need to be done now, but would really like to see the debouncing API integrated into the matrix here: 
https://docs.qmk.fm/#/feature_debounce_type